graylog2-web-interface/packages/graylog-web-plugin/package.json

```diff
@@ -66,7 +65,6 @@
   },
   "devDependencies": {
     "@babel/cli": "7.16.8",
-    "@babel/core": "7.16.12",
     "babel-plugin-add-module-exports": "^1.0.4"
```

Comment:
Do we really need the dependency `babel-plugin-add-module-exports` here as well?

Reply:
Ok, I just was that we need it here as well.